__tests__/index-test.js

```diff
@@ -36,6 +37,7 @@ class componentName extends Component {
 export default componentName;
 `,
     {
+      filename: "./filename-test.js",
```

Comment:
To be sure: is this an attribute that is set in Babel config or can it be inferred during the compilation phase by the plugin visitor?